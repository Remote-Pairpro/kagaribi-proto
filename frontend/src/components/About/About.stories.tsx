import { storiesOf } from "@storybook/react";
import * as React from "react";
import { About } from "./About";

storiesOf("About", module).add("default", () => {
    return <About />;
});
