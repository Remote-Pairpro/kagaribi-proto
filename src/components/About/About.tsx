import * as React from "react";

export const About = () => (
    <div>
        <h1>About</h1>
        <p>Kagaribi はあなたの生き方を照らします。</p>
    </div>
);

export default About;
