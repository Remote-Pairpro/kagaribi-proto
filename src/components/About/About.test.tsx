import { render, shallow } from "enzyme";
import React from "react";
import renderer from "react-test-renderer";
import About from "./About";

describe("<About />", () => {
    it("example", () => {
        const wrapper = shallow(<About />);
        expect(wrapper.find(".hoge").length).toBe(0);
    });

    it("should have description text", () => {
        const wrapper = render(<About />);
        expect(wrapper.text()).toContain("About");
        expect(wrapper.text()).toContain(
            "Kagaribi はあなたのスケジュールを照らし、調整を容易にします。"
        );
    });

    it("should renders correctly", () => {
        const tree = renderer.create(<About />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
