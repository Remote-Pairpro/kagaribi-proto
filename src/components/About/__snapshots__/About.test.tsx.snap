// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should renders correctly 1`] = `
<div>
  <h1>
    About
  </h1>
  <p>
    Kagaribi はあなたのスケジュールを照らし、調整を容易にします。
  </p>
</div>
`;
